packages/@ama-sdk/client-angular/src/api-angular-client.ts

```diff
@@ -159,10 +165,10 @@ export class ApiAngularClient implements ApiClient {
       exception = new EmptyResponseError(e.message || 'Fail to Fetch', undefined, { apiName, operationId, url, origin });
     }
 
-    // eslint-disable-next-line no-console
+    // eslint-disable-next-line no-console -- `console.error` is supposed to be the default value if the `options` argument is not provided, can be removed in Otter v12.
     const reviver = getResponseReviver(revivers, response, operationId, { disableFallback: this.options.disableFallback, log: console.error });
```

Comment:
Careful if we pass the logger to communicate with the teams before: there are a lot of logs that can be hidden behind this one

Reply:
after discussion text is okay